### PR TITLE
integrated eeg dataset for TUH

### PR DESCRIPTION
**Pull request by khaledsaab**

### CODE REVIEW

**Comment by seyuboglu** [comment]
Thanks for adding this! Instead of including all the files in the meerkat repo, can we provide a function that just downloads it from https://www.isip.piconepress.com/projects/tuh_eeg/html/downloads.shtml? This way average meerkat users won't have to download these files when cloning. We could do something similar to `meerkat.contrib.imagenette.download_imagenette`

**Comment by krandiash** [comment]
I was just literally about to comment this @seyuboglu

**Comment by khaledsaab** [comment]
Great suggestion. I'll work on that and push the updates

File: meerkat/contrib/eeg/data_utils.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            # time step size may not be 1-sec
+            start_time_step = int(np.floor(start_t_sec / time_step_size))
+            end_time_step = int(np.ceil(end_t_sec / time_step_size))
+
+            seizure_labels[start_time_step:end_time_step] = 1
+
+    return eeg_clip, seizure_labels, is_seizure
+
+
+def getSeizureTimes(file_name):
```
Comment (seyuboglu) [inline meerkat/contrib/eeg/data_utils.py:91]: nit: snake case for functions

**Review by seyuboglu — CHANGES_REQUESTED** [review]
A couple of nits, but otherwise lgtm once we remove the data files

**Comment by khaledsaab** [comment]
In the latest commit, I've removed unnecessary aux files and integrated necessary code to produce them if a meerkat user is interested in EEG. I also provided code and instructions for a meerkat user to download the EEG dataset, and do the necessary preprocessing steps. I'll be available to work through any bugs/errors that arise.

**Comment by krandiash** [comment]
Looks great, thanks!

**Pull request merged**